inst/tests/tests.Rraw

```diff
@@ -18732,3 +18734,46 @@ x = list(
 N = 5e4
 y = x[rep(1:5, N)]
 test(2268, rbindlist(y, fill=TRUE), rbindlist(x, fill=TRUE)[rep(1:5, N)])
+
+# rbind with vectors with class attributes #5309
```

Comment:
All of this uses `rbind()`, which test `rbindlist()` implicitly, but that's somewhat of an implementation detail, so let's add at least some tests on `rbindlist()` directly.

Also I don't see any test involving the `AsIs` logic that is included here.

Reply:
I added an comment to mark the `AsIs` cases better and also included the `rbindlist` there